docs/examples/QAOAFromPyQuil.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Max-Cut QAOA on QCS with Qiskit\n",
     "\n",
-    "> ℹ️ Originally written to demonstrate pyQuil, this notebook demonstrates how to take existing pyQuil work and run it on a Rigetti backend using Qiskit.\n",
+    "> ℹ️ Originally written to demonstrate pyQuil, this notebook shows how to take existing pyQuil work and run it on a Rigetti backend using Qiskit.\n",
```

Comment:
This is a bit confusing IMO. There's no link to the original, so the first part lacks context. And the second part seems to imply that the following notebook will show how one takes existing pyQuil and reworks it to use Qiskit but there's no comparison between the two.

I would link to the original, and say that this notebook is the Qiskit version.

Reply:
Yeah, I'm not sure how to link to it, as it's another notebook in JupyterHub...

Reply:
We'll remove this example.